src/app/visual/action-button/action-button-visual.component.html

```diff
@@ -18,71 +21,62 @@
       Start from scratch and fine-tune with filters
     </sky-action-button-details>
   </sky-action-button>
+  <br>
```

Comment:
The `<br>` is needed for the implementation?

Reply:
No, it's just here so manual testing is easier on the eye and the visual tests have some negative space to capture hover states / borders.